messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/CommonAttributes.java

```diff
@@ -100,6 +100,7 @@
     AttributeDefinition CONSUMER_COUNT = create("consumer-count", INT)
             .setStorageRuntime()
             .setRequired(false)
+            .addFlag(COUNTER_METRIC)
```

Comment:
it's a gauge (number of consumers can go up and down) as a counter is only monotonically increasing